### PR TITLE
REL-4044: Update partner calibration overheads in PIT

### PR DESCRIPTION
**Pull request by cquiroz**

### CODE REVIEW

File: bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/overheads/Overheads.scala [hunk 1]
```diff
@@ -13,66 +13,33 @@ import scala.annotation.tailrec
...
-  private case class SimpleOverheads(partnerOverheadFraction: Double) extends Overheads
+  private case class SimpleOverheads(partnerOverheadFraction: Double) extends Overheads {
+    def calculate(progTime: TimeAmount): ObservationTimes = {
+      val progTimeHrs = progTime.toHours.value
+      val partTimeHrs = progTimeHrs * partnerOverheadFraction
+      ObservationTimes(TimeAmount(progTimeHrs, TimeUnit.HR), TimeAmount(partTimeHrs, TimeUnit.HR))
+    }
+  }
+  private case class TelluricOverheads(telluricTime: TimeAmount) extends Overheads {
+    val VisitLength = 1.5
```
Comment (swalker2m) [inline bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/overheads/Overheads.scala:35]: We're saying, I think, that we need a telluric for every 1.5 hours (or fraction thereof) of science?  If so, maybe `ScienceHoursPerTelluric` would be a better name, idk.  I was initially confused by the name.

**Review by swalker2m — APPROVED** [review]
Looks good, fewer changes than I expected for this.  :+1:

File: bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/overheads/Overheads.scala (hunk 1 above)
Comment (cquiroz) [reply bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/overheads/Overheads.scala:35]: Sounds good. I'll also indicate the units

File: bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/overheads/Overheads.scala [hunk 2]
```diff
@@ -31,11 +31,11 @@ object Overheads extends (BlueprintBase => Option[Overheads]) {
       ObservationTimes(TimeAmount(progTimeHrs, TimeUnit.HR), TimeAmount(partTimeHrs, TimeUnit.HR))
     }
   }
-  private case class TelluricOverheads(telluricTime: TimeAmount) extends Overheads {
-    val VisitLength = 1.5
+  private case class ScienceHoursPerTelluric(telluricTime: TimeAmount) extends Overheads {
+    val VisitLength = TimeAmount(1.5, TimeUnit.HR)
```
Comment (swalker2m) [inline bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/overheads/Overheads.scala:35]: Oh, it was actually the `VisitLength` name itself I was confused about.  Maybe I'm just generally confused though.

Comment (cquiroz) [reply bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/overheads/Overheads.scala:35]: 😄 Te name `VisitLengh` comes directly from the original ticket

Comment (swalker2m) [reply bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/overheads/Overheads.scala:35]: Ok, I don't think they care what we call our internal variables but maybe it's best to match what they put in the task.  I think I'd put the class name back to `TelluricOverheads` though.  Sorry for the hassle!  It looks good. :+1:

Comment (swalker2m) [reply bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/overheads/Overheads.scala:35]: `NIRSpectroscopyOverheads` seems good.

**Pull request merged**